custom/icds_reports/reports/fact_sheets.py

```diff
@@ -662,8 +678,13 @@ def get_data(self):
                         row['data'].append({'html': 0})
 
                     if 'average' in row:
-                        data = self.get_data_for_national_aggregatation(
-                            row['data_source']
+                        data = get_data_for_national_aggregatation(
+                            self.config['domain'],
+                            self.config['previous_month'].strftime('%Y-%m-%d'),
```

Comment:
we have to convert to string and back because quickcache can't handle dates?

Reply:
yes thats right